src/math-translator.js

```diff
@@ -272,6 +277,8 @@ function translateMath(math, lang) {
  */
 function normalizeTranslatedMath(math, lang) {
 
+    if (lang === 'en') return math;
```

Comment:
Why would this be called for English?

Reply:
It's called from `getMapping()`, which is itself called by `createTemplate` to get English to English math mapping. 

https://github.com/Khan/translation-assistant/blob/61ec0a177137fb63e47379eebf4b04b65653628a/src/translation-assistant.js#L362